src/vr_session.hpp

```diff
@@ -7,7 +7,7 @@
 
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wnon-virtual-dtor"
-#include "../subprojects/openvr/headers/openvr.h"
+#include "openvr.h"
```

Comment:
Nit: can we use `<openvr.h>` to make it clearer it's not a local include anymore?